### ISSUE TITLE
Exeption on AmazonSQSExtendedClient.deleteMessage

### ISSUE THREAD

**Issue by nukede** [awslabs/amazon-sqs-java-extended-client-lib#12]
Just testing the sample App and got an error: 

```
Exception in thread "main" java.lang.AbstractMethodError: com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.deleteMessage(Lcom/amazonaws/services/sqs/model/DeleteMessageRequest;)Lcom/amazonaws/services/sqs/model/DeleteMessageResult;
    at com.di.maven.SQSExtendedClient.main(SQSExtendedClient.java:105)
    at com.di.maven.App.main(App.java:46)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
```

Line: 105
`sqsExtended.deleteMessage(new DeleteMessageRequest(myQueueUrl, messageReceiptHandle));`

Maven's dependecy: 

```
`dependency>
      <groupId>com.amazonaws</groupId>
      <artifactId>aws-java-sdk</artifactId>
      <version>1.11.14</version>
    </dependency>
    <dependency>
      <groupId>com.amazonaws</groupId>
      <artifactId>amazon-sqs-java-extended-client-lib</artifactId>
      <version>1.0.0</version>
    </dependency>
```

Any solutions?

**Comment by nukede** [comment]
Found out that the library is out-dated.
Requirements is: 
`    <dependency>
      <groupId>com.amazonaws</groupId>
      <artifactId>aws-java-sdk</artifactId>
      <version>1.10.22</version>
    </dependency>`

Or maybe the example on this (page)[https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/s3-messages.html] is out-dated?

**Comment by mvigneshtuty** [comment]
@nukede i'm having the same issue. When using sdk version 1.10.22 it ends up in below error message.

how you managed to solve this? any suggestions?

```
Exception in thread "main" com.amazonaws.AmazonServiceException: Failed to get the S3 object which contains the message payload. Message was not received. (Service: null; Status Code: 0; Error Code: null; Request ID: null)
	at com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.getTextFromS3(AmazonSQSExtendedClient.java:1010)
	at com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.receiveMessage(AmazonSQSExtendedClient.java:341)
	at org.nfjs.aws.sqs.SQSExtendedClient.receiveMessage(SQSExtendedClient.java:102)
	at org.nfjs.aws.sqs.SQSExtendedClient.main(SQSExtendedClient.java:64)
Caused by: com.amazonaws.services.s3.model.AmazonS3Exception: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint. (Service: Amazon S3; Status Code: 301; Error Code: PermanentRedirect; Request ID: 866529B4BDD0D8F2), S3 Extended Request ID: YfWmngETRt2PCAxtz7VvGpa28CYWSf9JQdNVETtgufLY2TGsSMqHHY2XA8LrazkdsxUomCJ1QmI=
	at com.amazonaws.http.AmazonHttpClient.handleErrorResponse(AmazonHttpClient.java:1182)
	at com.amazonaws.http.AmazonHttpClient.executeOneRequest(AmazonHttpClient.java:770)
	at com.amazonaws.http.AmazonHttpClient.executeHelper(AmazonHttpClient.java:489)
```

**Comment by kuba-aws** [comment]
Fixed in the 1.0.1 release.

**Comment by govardhanaraoganji** [comment]
@nukede nd @kuba-aws   I'm also facing same issue

- Using 
aws-sdk-s3 : 1.11.59
amazon-sqs-java-extended-client-lib: 1.0.1 & 1.0.2
 
or is there any latest version ?

```
ERROR 7258 --- [refetchThread-3] c.a.s.j.AmazonSQSMessagingClientWrapper  : AmazonServiceException: receiveMessage. RequestId: null
HTTPStatusCode: 0 AmazonErrorCode: null

com.amazonaws.AmazonServiceException: Failed to get the S3 object which contains the message payload. Message was not received. (Service: null; Status Code: 0; Error Code: null; Request ID: null)
	at com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.getTextFromS3(AmazonSQSExtendedClient.java:1180) ~[amazon-sqs-java-extended-client-lib-1.0.2.jar:na]
	at com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.receiveMessage(AmazonSQSExtendedClient.java:353) ~[amazon-sqs-java-extended-client-lib-1.0.2.jar:na]
	at com.amazon.sqs.javamessaging.AmazonSQSMessagingClientWrapper.receiveMessage(AmazonSQSMessagingClientWrapper.java:351) ~[amazon-sqs-java-messaging-lib-1.0.5.jar:na]
	at com.amazon.sqs.javamessaging.SQSMessageConsumerPrefetch.getMessages(SQSMessageConsumerPrefetch.java:262) ~[amazon-sqs-java-messaging-lib-1.0.5.jar:na]
	at com.amazon.sqs.javamessaging.SQSMessageConsumerPrefetch.run(SQSMessageConsumerPrefetch.java:221) ~[amazon-sqs-java-messaging-lib-1.0.5.jar:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: com.amazonaws.services.s3.model.AmazonS3Exception: The specified bucket does not exist (Service: Amazon S3; Status Code: 404; Error Code: NoSuchBucket; Request ID: null; S3 Extended Request ID: null)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1660) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1324) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1074) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:745) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:719) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:701) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:669) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:651) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:515) ~[aws-java-sdk-core-1.11.415.jar:na]
	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4443) ~[aws-java-sdk-s3-1.11.415.jar:na]
	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4390) ~[aws-java-sdk-s3-1.11.415.jar:na]
	at com.amazonaws.services.s3.AmazonS3Client.getObject(AmazonS3Client.java:1427) ~[aws-java-sdk-s3-1.11.415.jar:na]
	at com.amazon.sqs.javamessaging.AmazonSQSExtendedClient.getTextFromS3(AmazonSQSExtendedClient.java:1176) ~[amazon-sqs-java-extended-client-lib-1.0.2.jar:na]
	... 7 common frames omitted
```
